app/assets/js/components/UI/Dialog/Dialog.styled.jsx

```diff
@@ -43,10 +43,20 @@ const DialogTitle = styled(Dialog.Title, {
   fontWeight: "700",
 });
 
+const DialogFooter = styled(Dialog.Footer, {
```

Comment:
```suggestion
const DialogFooter = styled("footer", {
```

`Dialog.Footer` is not an export from Radix UI's `Dialog` (https://www.radix-ui.com/primitives/docs/components/dialog#anatomy), so you might run into a compilation error at worst, warning at best.